labs-ios-starter/Helpers/GooglePlacesAPIHelpers.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  GooglePlacesAPIHelpers.swift
+//  labs-ios-starter
+//
+//  Created by Conner on 2/18/21.
+//  Copyright © 2021 Spencer Curtis. All rights reserved.
+//
+
```

Comment:
Good job on the clean code, and great error messages. These messages are all distinct and will definitely notify us of where the error occured.

Reply:
Thanks!